README.md

```diff
@@ -84,6 +84,10 @@ In rare cases, you may want to also copy each dependency into the build product
 
 If you’ve modified your [Cartfile][], or you want to update to the newest versions of each framework (subject to the requirements you’ve specified), simply run the `carthage update` command again.
 
+### Transitive dependencies
```

Comment:
Can we find clearer wording than “transitive dependencies”? I think this will be needlessly opaque and off-putting; it makes it sound more complicated than it is.